styles.css

```diff
@@ -121,13 +238,146 @@ p {
   display: flex;
   position: relative;
   flex-direction: column;
-  padding-top: 20%;
+  padding-top: 5%;
+}
+
+.works #web_img {
+  display: none;
 }
 
 .works h2 {
   padding: 20px;
 }
 
+.card_content-web {
+  display: none;
+}
+
+#mobile-title {
+  display: flex;
+}
+
+@media screen and (min-width: 800px) {
+  .card_content-web {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+
+  #mobile-title {
+    display: none;
+  }
+}
+
```

Comment:
- [ ] Probably, in the cause of implementing the changes required by the previous reviewer, you broke some layout of your page. The project image here actually occupied more than you intended and as a result created extra space to the right side of the whole page. That alone has distorted many other layout of the page. Kindly look into how you can adjust it to fit 100% width of your page without extra space.

### Your Design
[project-layout.webm](https://user-images.githubusercontent.com/26597432/207275181-52865cd2-08a7-4c9a-ae03-af25a8f4f1cc.webm)

### Figma Design
![figma-projects](https://user-images.githubusercontent.com/26597432/207275403-53454e5e-55ff-4002-b5d2-0d94fc2a2090.png)